### PR TITLE
Add deprecated method to extend_schema decorator

### PR DESCRIPTION
**Pull request by MissiaL**

### CODE REVIEW

File: drf_spectacular/openapi.py [hunk 1]
```diff
@@ -231,6 +235,10 @@ def get_operation_id(self, path, method):
 
         return '_'.join(tokenized_path + [action])
 
+    def get_deprecated(self, path, method):
```
Comment (tfranzel) [inline drf_spectacular/openapi.py]: i think ``is_deprecated`` is a more fitting name

File: drf_spectacular/utils.py [hunk 2]
```diff
@@ -114,6 +116,11 @@ def get_description(self, path, method):
                     return description
                 return super().get_description(path, method)
 
+            def get_deprecated(self, path, method):
```
Comment (tfranzel) [inline drf_spectacular/utils.py]: same as above

File: drf_spectacular/openapi.py [hunk 3]
```diff
@@ -231,6 +235,10 @@ def get_operation_id(self, path, method):
...
+        """ override this for custom behaviour """
+        return []
```
Comment (tfranzel) [inline drf_spectacular/openapi.py]: ``False`` is more explicit

**Comment by MissiaL** [comment]
Fixed. Please check

**Comment by tfranzel** [comment]
awesome! thank you

**Pull request merged**